### PR TITLE
Docs/install tutorials

### PR DESCRIPTION
**Pull request by Anafabula**
Docs: Install page, tutorials page & extend common errors page.

Feat: Make the application runnable without VSCode
Feat: Add fixture for Widget settings

Fix: Missing icons in prod mode

### CODE REVIEW

**Comment by jarkenau** [comment]
It's nice that you included the start of the application into the dev-container, but I'm not sure if it really makes sense. Dev people probably don't want an automatic startup, and for a standalone server the code is never copied into the Docker image. 

I would go for an approach like SEEEP does, have a base image for the dev-container and then a server image where the code is copied into https://github.com/agri-gaia/seerep/tree/main/docker. Ofc we would also need a separate docker compose file.

File: docs/getting-started/errors.md [hunk 1]
```diff
@@ -38,3 +50,34 @@ Docker Compose version v2.5.0
...
+5050 (for the frontend in preview mode) to be available.
+
+<figure markdown>
+  ![Docker port error](../imgs/docker-port-error.png){width=700px}
+  <figcaption> Docker port error </figcaption>
+</figure>
+
+If one of these ports is already in use, the launch may fail with an error like the one shown above.
+
+Running docker containers for example may be using one or more of the required ports.
```
Comment (jarkenau) [inline docs/getting-started/errors.md:67]:
```suggestion
Already running docker containers, may use the required ports.
```

File: docs/getting-started/installation.md [hunk 2]
```diff
@@ -1 +1,107 @@
...
+2. Open the repository in VSCode.
+
+   ```sh
+   code .
+   ```
+
+3. Press `F1` or `CTRL+SHIFT+P` and enter `Remote-Containers: Reopen Folder in Container`
+
+   This downloads a docker container for the development of the PlantMap application and
+   starts the needed [PostgreSQL](https://www.postgresql.org/) and RabbitMQ. Additionally,
```
Comment (jarkenau) [inline docs/getting-started/installation.md:55]:
```suggestion
   starts the needed [PostgreSQL](https://www.postgresql.org/) and [RabbitMQ}(https://www.rabbitmq.com/). Additionally,
```

File: docs/getting-started/tutorials.md [hunk 3]
```diff
@@ -1 +1,108 @@
...
+
+### Crops :seedling:
+
+<figure markdown>
+  ![Crops tab](../imgs/crops-tab.png){width=700px}
+  <figcaption> Crops tab </figcaption>
+</figure>
+
+The crops tab consists of a table showing the beds of the current garden.
+Clicking on one of the beds shows the crops of the clicked bed in the table instead.
```
Comment (jarkenau) [inline docs/getting-started/tutorials.md:55]:
```suggestion
Clicking on one of the beds shows it's crops.
```

File: docs/getting-started/tutorials.md [hunk 4]
```diff
@@ -1 +1,108 @@
...
+
+To the right side of the table is a map of the current garden.
+Green markers show the location of all the beds listed in the table and
+hovering over one of them highlights the corresponding row.
+Clicking on one of the bed markers has the same effect as clicking on a bed in the table and
+shows the crops of the clicked bed.
```
Comment (jarkenau) [inline docs/getting-started/tutorials.md:61]:
```suggestion
Clicking on one of the bed markers has the same effect as clicking on a bed in the table, the crops are shown.
```

File: docs/getting-started/tutorials.md [hunk 5]
```diff
@@ -1 +1,108 @@
...
+
+The 3D tab has the same table and map as the [Crops](#crops) tab, the map is just larger.
+Just like the Crops tab, beds and crops each have buttons in the `3D` column of the table.
+
+Clicking the button brings you to a 3D model of the bed.
+The model can be rotated with the left and moved with the right mouse button.
+The scroll wheel can be used to zoom.
+
+If you clicked on the `3D` button for a crop, an overlay will be shown,
+framing the crop and providing additional information about it.
```
Comment (jarkenau) [inline docs/getting-started/tutorials.md:84]:
```suggestion
If you clicked on the `3D` button for a crop, an overlay with information will be shown. 
```

**Review by jarkenau — COMMENTED** [review]
I would add some additional screenshots in the `Login` and `Sign up` section of the tutorials page, otherwise a new user is kind of lost. Also consider adding arrows or boxes in the pictures to highlight important parts. Otherwise, it looks good to me :+1:.

@dobro929 can judge the frontend stuff :sweat_smile:

**Review by jarkenau — APPROVED** [review]

**Review by dobro929 — APPROVED** [review]
everything looks good👍🏼

**Pull request merged**